Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -25,14 +25,15 @@ public protocol ProtobufBinaryCodableType: ProtobufTypePropertiesBase {
 
     static func setFromProtobufVarint(varint: UInt64, value: inout BaseType?) throws -> Bool
     static func setFromProtobufVarint(varint: UInt64, value: inout [BaseType]) throws -> Bool
```

Comment:
All of them throw if the input cannot be parsed for any reason.

Three of these have special requirements from the unknown enum handling.  It may be possible to rework that, but at least that detail is entirely isolated to the ProtobufBinary*.swift sources.